package-overrides/github/allpro/layout@1.4.3.json

```diff
@@ -9,7 +9,7 @@
     },
     "shim": {
         "jquery-ui-layout": {
-            "deps": "jquery"
+            "deps": ["jquery","jquery-ui"]
```

Comment:
We do need to explicitly state dependencies in the package.json as well as providing the shim config. The reason for this is that the `dependencies` ensures it is downloaded, while the shim just acts like a `require` statement. Note we also usually want to include full names in the package.json dependencies, like `github:components/jqueryui` the reason being that this way we can make backwards-incompatible changes to the registry itself without issue.